astrobase/lcmath.py

```diff
@@ -291,16 +305,22 @@ def sigclip_magseries(times, mags, errs,
     find = npisfinite(times) & npisfinite(mags) & npisfinite(errs)
     ftimes, fmags, ferrs = times[find], mags[find], errs[find]
 
-    # get the median and stdev = 1.483 x MAD
-    median_mag = npmedian(fmags)
-    stddev_mag = (npmedian(npabs(fmags - median_mag))) * 1.483
+    # get the center value and stdev 
+    if meanormedian == 'median':  # stddev = 1.483 x MAD
+        center_mag = npmedian(fmags)
+        stddev_mag = (npmedian(npabs(fmags - center_mag))) * 1.483
+    elif meanormedian == 'mean':
+        center_mag = npmean(fmags)
+        stddev_mag = npstddev(fmags)
+    else:
+        LOGERROR("Unrecognized meanormedian value given to sigclip_magseries: " + str(meanormedian))
```

Comment:
Better to use:
```python
LOGERROR("Unrecognized meanormedian value given to sigclip_magseries: %s " % meanormedian)
```
We should also deal with the weird `meanormedian` input here. I suggest changing the LOGERROR to LOGWARNING, setting `meanormedian` to 'median' and proceeding to process the LC after informing about the problem.